src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure implements AreaCalculetion {
+    private double firstLeg;
+    private double secondLeg;
+
+    public RightTriangle(double firstLeg, double secondLeg, String color, String name) {

```

Comment:
The constructor should not contain logic. It's better to create separate variables for `firstLeg` and `secondLeg` before passing them to the constructor for better code readability.